grafana/001-login-oauth-use-oauth2-exchange.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/pkg/api/login_oauth.go b/pkg/api/login_oauth.go
```

Comment:
Isn't it better to have a fork of this repo within maistra org and push those changes there? Just like we do with istio?

Reply:
This package is based on grafana packaging work done at Red Hat, there's a chance we'll be able to reuse their work without any changes at some point. I'd like to try keeping things close to their work, at least for the time being.